scripts/regrid.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# load packages
+import os, glob, sys, re, argparse
+import xesmf as xe
+import xarray as xr
+import numpy as np
+from netCDF4 import Dataset
+import pandas as pd
+import cartopy.crs as crs
+# internal export
+sys.path.append('/project/p/peltier/edmundn/climate_normals_Py/scripts/')
+from Climate_normals import get_nth_word_custom_delimiter, build_parameter
+from wrf import (getvar, ALL_TIMES, to_np,get_cartopy, latlon_coords,Constants)
+# Define global setting
+xr.set_options(keep_attrs=True)
+# geo file for wrfna24 grid
+geo_file = '/scratch/p/peltier/mahdinia/wrf/MPIESM12HR_HS_na24_conus43-flk_1979/geo_em.d01.nc'
+
```

Comment:
Add grid folder here, but make it dependent on and env var:
`data_root = os.getenv("DATA_ROOT") 
if data_root is None:
    raise ValueError("No DATA_ROOT defined")
grid_folder = f"{data_root}/grids/"`

Reply:
We then find grids like so: `grid_file = f"{grid_folder}/{grid_name}.nc"`